src/helper/soroban-rpc/index.ts

```diff
@@ -386,14 +387,43 @@ const isTokenSpec = (spec: Record<string, any>) => {
   return true;
 };
 
-// We can know when a token contract is a SAC instance by getting the name field, and checking for
-// the name to follow the pattern "assetId:issuer".
-const isSacIssuer = (name: string) => {
-  const issuerParts = name.split(":");
-  const [_, issuer] = issuerParts;
-  if (issuerParts.length > 1 && isPubKey(issuer)) {
-    return true;
+const isSacContractExecutable = async (
```

Comment:
ended up not using this but wanted to leave it since it's generally useful and could be a nice one to pluck out to the wallet sdk. cc @CassioMG @piyalbasu